.github/labeler.yml

```diff
@@ -0,0 +1,16 @@
+# https://github.com/marketplace/actions/labeler
+
+as-framework:
+  - '**/*'
+
+javascript-sdk:
+  - sdk/**/*.js
+  - codegen/js-gen/**/*
+  - samples/js/**/*
+
+typescript-sdk:
+  - sdk/**/*.ts
```

Comment:
It's not that simple to distinguish, as some of the core code is in TS files. The TypeScript SDK is the JavaScript SDK plus type definitions, which are currently defined as jsdoc, either in the JS files or in the the .jsdoc files when the source is in TS.